lib/elixir/lib/string.ex

```diff
@@ -276,6 +276,10 @@ defmodule String do
   Empty strings are only removed from the result if the
   `trim` option is set to `true` (default is `false`).
 
+  When the pattern used is a regular expression, the string will be
+  split using `Regex.split/3`. This means that it will accept
```

Comment:
Let's say "This means that <strike>it</strike>this function <strike>will accept</strike>acceots [...]", wdyt? We usually use present verbs and I feel "it" is not superclear here. :)